build.sbt

```diff
@@ -85,18 +86,20 @@ lazy val actor = akkaModule("akka-actor")
   .settings(Dependencies.actor)
   .settings(OSGi.actor)
   .settings(AutomaticModuleName.settings("akka.actor"))
-  .settings(unmanagedSourceDirectories in Compile += {
-    val ver = scalaVersion.value.take(4)
-    (scalaSource in Compile).value.getParentFile / s"scala-$ver"
-  })
+  .settings(addSourceDirScalaBin())
   .settings(VersionGenerator.settings)
   .enablePlugins(BoilerplatePlugin)
+  .dottySupport()
 
 lazy val actorTests = akkaModule("akka-actor-tests")
   .dependsOn(testkit % "compile->compile;test->test")
   .settings(Dependencies.actorTests)
   .enablePlugins(NoPublish)
   .disablePlugins(MimaPlugin, WhiteSourcePlugin)
+  .dottySupport()
+  // https://github.com/lampepfl/dotty/issues/8599
+  // TODO remove when the issue is fixed
+  .settings(excludeSourcesIfDotty(Seq("akka/actor/StashJavaAPITestActors.java", "akka/actor/StashJavaAPI.java")))
```

Comment:
can this from config file?

Reply:
I have moved this to another commit that is in [dotty-wip](https://github.com/ohze/akka/blob/dotty-wip/project/DottySupport.scala#L46) branch only (not in `dotty-syntax-compat` branch of this PR)